app/views/shared/_filters.html.erb

```diff
@@ -11,20 +9,16 @@
       <div class='form-row'>
         <%= f.select :filter_tag_ids,
           grouped_options_for_select(tag_options, filter_tag_ids),
-          { label: t('filters.tags'), wrapper: { class: 'col-md-5' } },
+          { label: t('filters.tags'), wrapper: { class: 'col-md-8' } },
           { class: 'filter',
             multiple: true, 'data-actions-box': true, 'data-use-select2': true } %>
 
-        <%= f.date_select :filter_start_time,
-          { label: t('filters.start_time'), wrapper: { class: 'col-md-3' },
-            order: %i[month year], default: filter_start_time },
-          class: 'filter'
-        %>
-
-        <%= f.date_select :filter_end_time,
-          { label: t('filters.end_time'), wrapper: { class: 'col-md-3' },
-            order: %i[month year], default: filter_end_time },
-          class: 'filter' %>
+        <%= f.text_field :filter_date_range,
+          { #value: @presenter.filter_date_range,
+            label: t('filters.date_range'),
+            wrapper: { class: 'col-md-3' },
+            class: 'filter', autocomplete: 'off', 'data-use-litepicker': true,
```

Comment:
If we add readonly: true here, it prevents the user from clearing the text_field contents (and from entering any other invalid data) which can lead to this error:

![Screenshot 2021-03-10 at 08 11 05](https://user-images.githubusercontent.com/40230316/110599528-dd13fd00-817a-11eb-9cd3-ee8eb281d998.png)